abis/WrappedToken.json

```diff
@@ -0,0 +1,385 @@
+[
```

Comment:
Just to confirm, did you pull this ABI from gm-contracts or from the actual WETH or WFTM contract on ETH/FTM mainnet? If this is from gm-contracts it is more accurate to pull it from mainnet instead

Reply:
pull it from mainnet then

Reply:
will pull it from mainnet then

Reply:
will pull it from mainnet then